octobot_trading/personal_data/trades/channel/trades_updater.py

```diff
@@ -69,6 +69,12 @@ async def fetch_and_push(self):
             trades: list = await self.channel.exchange_manager.exchange.get_my_recent_trades(
                 symbol=symbol,
                 limit=self.MAX_OLD_TRADES_TO_FETCH)
+            if not trades:
+                # on some exchanges, recent trades are only fetching very recent trade. also try closed orders
```

Comment:
Why no managing it in the ccxt exchange (or exchange tentacle)? I don't understand why we should fix exchanges behaviors here

Reply:
right, i'll change it